src/NuGetPackageVerifier/Program.cs

```diff
@@ -169,6 +169,16 @@ public static int Main(string[] args)
 
                     processedPackages.Add(package);
                 }
+
+                foreach (var issue in issueProcessor.IssuesToIgnore)
+                {
+                    // TODO: Don't show this for rules that we don't run.
+                    logger.LogWarning(
+                        "Unnecessary exclusion in {0}\n\tIssue: {1}\n\tInstance: {2}\n",
```

Comment:
Yuck, newlines in a log message :smile: But at least change `\n` to `Environment.NewLine`?